include/libpldm/rde.h

```diff
@@ -539,6 +548,81 @@ int decode_rde_multipart_send_resp(const struct pldm_msg *msg,
 				   size_t payload_length,
 				   uint8_t *completion_code,
 				   uint8_t *transfer_operation);
+/**
+ * @brief Encode RDEMultipartReceive request.
+ *
+ * @param[in] instance_id - Message's instance id.
+ * @param[in] data_transfer_handle - A handle to uniquely identify the chunk
+ * of data to be retrieved.
+ * @param[in] operation_id - Identification number for this operation.
+ * @param[in] transfer_operation - The portion of data requested for the
+ * transfer.
+ * @param[out] msg - Request will be written to this.
+ * @return pldm_completion_codes.
+ */
+int encode_rde_multipart_receive_req(uint8_t instance_id,
+				     uint32_t data_transfer_handle,
+				     uint16_t operation_id,
+				     uint8_t transfer_operation,
+				     struct pldm_msg *msg);
+
+/**
+ * @brief Decode RDEMultipartReceive request.
+ *
+ * @param[in] msg - Request message.
+ * @param[in] payload_length - Length of request message payload.
+ * @param[out] data_transfer_handle - A handle to uniquely identify the
+ * chunk of data to be retrieved.
+ * @param[out] operation_id - Identification number for this operation.
+ * @param[out] transfer_operation - The portion of data requested for the
+ * transfer.
+ * @return pldm_completion_codes.
+ */
+int decode_rde_multipart_receive_req(const struct pldm_msg *msg,
+				     size_t payload_length,
+				     uint32_t *data_transfer_handle,
+				     uint16_t *operation_id,
+				     uint8_t *transfer_operation);
+/**
+ * @brief Encode RDEMultipartReceive response.
+ *
+ * @param[in] instance_id - Message's instance id.
+ * @param[in] completion_code - PLDM completion code.
+ * @param[in] transfer_flag - The portion of data being sent to MC.
+ * @param[in] next_data_transfer_handle - A handle to uniquely identify the
+ * next chunk of data to be retrieved.
+ * @param[in] data_length_bytes - Length of the payload.
+ * @param[in] add_checksum - Indicate whether the payload needs to include
+ * the provided checksum.
+ * @param[in] data_integrity_checksum - Checksum.
+ * @param[in] data - Pointer to the data.
+ * @param[out] msg - Response message will be written to this.
+ * @return pldm_completion_codes.
+ */
+int encode_rde_multipart_receive_resp(
+	uint8_t instance_id, uint8_t completion_code, uint8_t transfer_flag,
+	uint32_t next_data_transfer_handle, uint32_t data_length_bytes,
+	bool add_checksum, uint32_t data_integrity_checksum,
```

Comment:
add_checksum is not spec defined right?better to handle interanlly and for now you can add to do if the support is not inplace.